src/core/plugins/view/root-injects.jsx

```diff
@@ -109,7 +109,11 @@ export const getComponent = (getSystem, getStore, getComponents, componentName,
   let component = getComponents(componentName)
 
   if(!component) {
-    getSystem().log.warn("Could not find component", componentName)
+    if (!process.env.NODE_ENV !== "production") {
+      // In the json schema rendering code, we optimistically query our system for a number of components.
+      // If the component doesn't exist, these warnings get thrown (in non-production mode).
+      getSystem().log.warn("Could not find component", componentName)
+    }
```

Comment:
Before we merge this @tim-lai, I'd like to go through the exercise of making sure that a consumer of Swagger UI that is in development is still able to see `getComponent` warnings. (They should, but it's certainly possible that something in our Webpack configs is transforming this value to be computed based on _our_ environment at build time, not theirs at runtime).

Roughly, here's how you'd do it:
1. Set up a project that is importing Swagger UI as a module, with a plugin that calls `getComponent` for a non-existing plugin. (My [swagger-ui-cra](https://github.com/shockey/swagger-ui-cra) repo is a bit outdated, but is a good starting point!)
2. Back in your Swagger UI folder, run `npm run build` while on this branch to generate fresh distribution assets that include your changes.
3. Use `npm link` to pull your freshly-generate assets into your project that consumes Swagger UI.
4. Run your project, and confirm that `getComponent` warnings are still visible!